src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+Microsoft.CodeAnalysis.VisualBasic.Syntax.SyntaxNodeRemover.New() -> Void
```

Comment:
Its a mix of both - majority of them are already shipped but untracked in public API files, but few are for new unshipped APIs. Let me open a separate bug for @jaredpar or @shyamnamboodiripad to go through these new implicit constructor entries and move them to shipped/unshipped appropriately.